nav2_theta_star_planner/src/theta_star_planner.cpp

```diff
@@ -189,6 +194,30 @@ nav_msgs::msg::Path ThetaStarPlanner::linearInterpolation(
   return pa;
 }
 
+rcl_interfaces::msg::SetParametersResult
+ThetaStarPlanner::dynamicParametersCallback(std::vector<rclcpp::Parameter> parameters)
+{
+  rcl_interfaces::msg::SetParametersResult result;
+  for (auto parameter : parameters) {
+    const auto & type = parameter.get_type();
+    const auto & name = parameter.get_name();
+
+    if (type == ParameterType::PARAMETER_INTEGER) {
+      if (name == name_ + ".how_many_corners") {
+        planner_->how_many_corners_ = parameter.as_int();
+      }
+    } else if (type == ParameterType::PARAMETER_DOUBLE) {
+      if (name == name_ + ".w_euc_cost") {
+        planner_->w_euc_cost_ = parameter.as_double();
+      } else if (name == name_ + ".w_traversal_cost") {
+        planner_->w_traversal_cost_ = parameter.as_double();
+      }
+    }
+  }
```

Comment:
I think this is missing `use_final_approach_orientation`

Reply:
fixed it